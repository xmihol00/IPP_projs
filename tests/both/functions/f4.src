.IPPcode21

DEFVAR GF@var
MOVE GF@var int@5

CALL f4a
EXIT int@0

LABEL f4a
WRITE GF@var
WRITE string@\010
CALL f4b
WRITE GF@var
WRITE string@\010

LABEL f4b
ADD GF@var GF@var GF@var
WRITE GF@var
WRITE string@\010
RETURN