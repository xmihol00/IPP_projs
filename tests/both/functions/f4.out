5
10
10
20
